docs/reference/connectors/gcl_writer.md

```diff
@@ -0,0 +1,239 @@
+---
+sidebar_label: gcl_writer (Google Cloud Logging - writer)
+sidebar_position: 1
+---
+
+# The `gcl_writer` Connector
+
+The GCL Writer connector integrates [Google Cloud Logging](https://cloud.google.com/logging) from
+the Google Cloud Platform [Operations Suite](https://cloud.google.com/products/operations). This connector
+allows json logs to be published to Cloud Logging.
+
+
+## Configuration
+
+```tremor
+define connector gcl_writer from gcl_writer
+with
+    config = {
+        "connect_timeout": 1000000, # defaults to 1 second
+        "request_timeout: 1000000   # defaults to 10 seconds
+    }
+```
```

Comment:
this is missing an `end` at the end, and it'd be nice to use `nonos::from_seconds()`

Reply:
Both 👍